implementation/src/main/java/io/smallrye/metrics/legacyapi/HistogramAdapter.java

```diff
@@ -22,6 +22,12 @@ class HistogramAdapter implements Histogram, MeterHolder {
 
     private final static int PRECISION;
 
+    final MeterRegistry registry;
+
+    HistogramAdapter(MeterRegistry registry) {
+        this.registry = registry;
+    }
+
```

Comment:
it doesn't look like you use `registry` in this class - is it needed?

Reply:
The https://github.com/smallrye/smallrye-metrics/issues/562 it noted that the Timer and Histogram had inconsistent constructors, and should be the same to make it consistent for ease of future updates. Wasn't sure why it needed to be consistent ;)

Reply:
I just wanted it to be consistent where it makes sense.  in this case, we should just not take the registry as a ctor parameter since we're not using it.